src/components/personkort/ledere/PersonKortVirksomhetLedere.tsx

```diff
@@ -62,38 +79,38 @@ export const PersonKortVirksomhetLederColumn = (
 };
 
 interface PersonKortVirksomhetLederRowProps {
-  leder: Leder;
-  isActive: boolean;
+  leder: NarmesteLederRelasjonDTO;
 }
 
 export const PersonKortVirksomhetLederRow = (
   personKortVirksomhetLederRowProps: PersonKortVirksomhetLederRowProps
 ) => {
-  const { leder, isActive } = personKortVirksomhetLederRowProps;
+  const { leder } = personKortVirksomhetLederRowProps;
+  const isActive = leder.status == NarmesteLederRelasjonStatus.INNMELDT_AKTIV;
```

Comment:
Er det meningen å bruke dobbel-equals her? Mulig det funker, men er vel anbefalt med trippel-equals.

Reply:
For mye Kotlin på meg nå kanskje, skal endre til trippe!